test1.js

```diff
@@ -3,3 +3,9 @@
 if(foo  === "bar") {
   console.log( 'hey' )
 }
+
+var foo3 = '' ;
+
+if(foo3  === "bar3") {
+  console.log( 'hey3' );
```

Comment:
Codacy has a fix for the issue: [Replace `·'hey3'·` with `"hey3"`](https://release.k8s.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=5797)
```suggestion
  console.log("hey3");
```